apps/ui/src/pages/applications/sections/institutional.tsx

```diff
@@ -0,0 +1,265 @@
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { zodResolver } from '@hookform/resolvers/zod';
+import { Col, Form, Row } from 'antd';
+import { createSchemaFieldRule } from 'antd-zod';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useOutletContext } from 'react-router';
+import * as z from 'zod';
+
+import SectionWrapper from '@/components/layouts/SectionWrapper';
+import InputBox from '@/components/pages/application/form-components/InputBox';
+import SelectBox from '@/components/pages/application/form-components/SelectBox';
+import SectionContent from '@/components/pages/application/SectionContent';
+import SectionFooter from '@/components/pages/application/SectionFooter';
+import SectionTitle from '@/components/pages/application/SectionTitle';
+import { ApplicationOutletContext } from '@/global/types';
+
+type FieldType = {
+	institutionalTitle: string;
+	institutionalFirstName: string;
+	institutionalLastName: string;
+	institutionalSuffix: string;
+	institutionalPrimaryAffiliation: string;
+	institutionalInstituteAffiliation: string;
+	institutionalProfileUrl: string;
+	institutionalPositionTitle: string;
+	institutionCountry: string;
+	institutionState: string;
+	institutionCity: string;
+	institutionStreetAddress: string;
+	institutionPostalCode: string;
+	institutionBuilding: string;
+};
+
+const schema = z.object({
+	institutionalTitle: z.string().min(1, { message: 'Required' }),
+	InstitutionalFirstName: z
+		.string()
+		.min(1, { message: 'Required' })
+		.max(15, { message: 'First name should be less than 15 characters' }),
+});
+
+const rule = createSchemaFieldRule(schema);
+
+const Institutional = () => {
+	const { t: translate } = useTranslation();
+	const { isEditMode } = useOutletContext<ApplicationOutletContext>();
+
+	const { handleSubmit, control } = useForm<FieldType>({
+		resolver: zodResolver(schema),
+	});
+
+	const onSubmit: SubmitHandler<FieldType> = (data) => {
+		console.log(data);
+	};
+
+	return (
+		<SectionWrapper>
+			<Form layout="vertical" onFinish={handleSubmit(onSubmit)}>
+				<SectionTitle
+					title={translate('institutional-section.title')}
+					text={[translate('institutional-section.description1')]}
+				/>
+				<SectionContent title={translate('institutional-section.section1')}>
+					<Row>
+						<Col xs={{ flex: '100%' }} md={{ flex: '100%' }} lg={{ flex: '25%' }}>
+							<SelectBox
+								label={translate('institutional-section.form.title')}
+								name="institutionalTitle"
+								control={control}
+								rule={rule}
+								options={[
+									{ value: 'dr', label: 'Dr' },
+									{ value: 'miss', label: 'Miss' },
+									{ value: 'mr', label: 'Mr' },
+									{ value: 'mrs', label: 'Mrs' },
+									{ value: 'ms', label: 'Ms' },
+									{ value: 'prof', label: 'Prof' },
+								]}
+								required
+								disabled={!isEditMode}
```

Comment:
The design has a placeholder here that says 'Select', not sure if we want to add one or not